src/guides/v2.3/graphql/mutations/place-order.md

```diff
@@ -73,8 +73,8 @@ Attribute |  Data Type | Description
 
 Attribute |  Data Type | Description
 --- | --- | ---
-`order_id` String | The unique ID that identifies the order
-`order_number` String | The unique ID that identifies the order
+`order_id` |String! | Deprecated.Use order_number instead.The unique ID that identifies the order
```

Comment:
Hi @VinothKumar361. Please, make sure that the number of whitespaces is proper for both lines

```suggestion
`order_id` | String! | Deprecated. Use order_number instead. The unique ID that identifies the order
```